src/main/java/com/bgsoftware/superiorskyblock/island/bank/SIslandBank.java

```diff
@@ -50,7 +51,7 @@ public SIslandBank(Island island){
 
     @Override
     public BigDecimal getBalance() {
-        return balance.get();
+        return balance.get().setScale(2, RoundingMode.HALF_DOWN);
```

Comment:
The idea of rounding the balance is great, but I think there can be a better solution - you should round the balance when it's set, and therefore it will also get saved and displayed rounded.
This should be tested of course, but it should work.

Reply:
Originally that was my idea as well, but then i realized that if we want to round the setter as  well, we would need to add rounding to SIslandBank#increaseBalance as well, since it acts as a private setter. So i figured it would be easier just to add rounding to the getter. Also there's no point of rounding the number twice (during get & set), getter rounding will do just fine. I can make the change if you really want to though.

Reply:
Due to potential issues with threads and synchronizations, there is no way to combine the methods, and therefore yes - I prefer it to be in both setters and have a tiny of duplication code

Reply:
I shall do that then.